.travis.yml

```diff
@@ -23,12 +23,13 @@ cache:
 
 env:
   global:
-    - STRIPE_MOCK_VERSION=0.7.0
+    - STRIPE_MOCK_VERSION=0.8.0
 
 go:
-  - 1.7
-  - 1.8
-  - 1.9
+  - "1.7"
+  - "1.8"
+  - "1.9"
+  - "1.10"
```

Comment:
Had to wrap the version in quotes otherwise it parses to 1.1. I wrapped the other versions for consistency.